boot/boot.go

```diff
@@ -284,18 +284,18 @@ type BootableSet struct {
 	Kernel     *snap.Info
 	KernelPath string
 
+	RecoverySystem string
```

Comment:
called like this I was expecting just a system label, but it's a dir (which is fine, other things are paths here) but then it should be called RecoverySystemDir for clarity I think